app/src/Command/SendAuthMailRequestCommand.php

```diff
@@ -0,0 +1,334 @@
+use Symfony\Component\Mailer\Mailer;
+use Symfony\Component\Mailer\Transport;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Process\Process;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use App\Repository\DomainRepository;
+use App\Amavis\DeliveryStatus;
+
+#[AsCommand(
+    name: 'agentj:send-auth-mail-token',
+    description: 'Send email with url token to validate email sender address',
+)]
+class SendAuthMailRequestCommand extends Command
+{
+
+    private MessageStatus $messageStatusError;
+    private MessageStatus $messageStatusAuthorized;
+   
+    public function __construct(
+        private EntityManagerInterface $em,
+        private MsgsRepository $msgsRepository,
+        private TranslatorInterface $translator,
+        private CryptEncryptService $cryptEncryptService,
+        #[Autowire(param: 'domain')]
+        public readonly string $agentjDomain,
+        #[Autowire(param: 'scheme')]
+        public readonly string $agentjDomainScheme,
+        #[Autowire(param: 'app.smtp-transport')]
+        public readonly string $smtpTransport,
+        #[Autowire(param: 'app.amavisd-release')]
+        public readonly string $amavisdRelease,
+        #[Autowire(param: 'app.domain_mail_authentification_sender')]
+        public readonly string $defaultSenderAdress,
+    ) {
+        $this->messageStatusError = $this->em->getRepository(MessageStatus::class)->find(MessageStatus::ERROR);
+        $this->messageStatusAuthorized = $this->em->getRepository(MessageStatus::class)->find(MessageStatus::AUTHORIZED);
+        parent::__construct();
+    }
+    
+
+    /**
+     * 
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output):int
+    {
+
+
+        $store = new SemaphoreStore();
+        $factory = new LockFactory($store);
+        $lock = $factory->createLock('msgs-send-mail-token', 1800);
+
+        if ($lock->acquire()) {
+
+            $messagesToHandle = $this->msgsRepository->searchMsgsToSendAuthRequest();
+            $userRepository = $this->em->getRepository(User::class);
+
+            foreach ($messagesToHandle as $message) {
+                $recipientsByDomain = [];
+
+                foreach ($message->getMsgrcpts() as $msgrcpt) {
+                    $maddr = $msgrcpt->getRid();
+
+                    if (!$maddr) {
+                        continue;
+                    }
+
+                    $user = $userRepository->findOneBy([
+                        'email' => $maddr->getEmailClear()
+                    ]);
+
+                    if ($user === null) {
+                        continue;
+                    }
+
+                    if ($user->getBypassHumanAuth() && $msgrcpt->getStatus() === null && $msgrcpt->getDs() !== DeliveryStatus::Pass) {
```

Comment:
```suggestion
                    if ($user->getBypassHumanAuth() && $msgrcpt->getStatus() === MessageStatus::UNTREATED && $msgrcpt->getDs() !== DeliveryStatus::Pass) {
```

Ou mieux, mais faudrait créer des fonctions dans `Msgrcpt` : 

```suggestion
                    if ($user->getBypassHumanAuth() && $msgrcpt->isUntreated() && !$msgrcpt->isDeliveryPassed()) {
```

Questions supplémentaires : 

- si le status !== "untreated", est-ce qu’on a vraiment besoin de tester le delivery status ?
- si le status !== "untreated", est-ce qu’on a encore besoin d’envoyer l’email d’authentification ?